doc/source/10min.rst

```diff
@@ -425,7 +426,7 @@ String Methods
 Series is equipped with a set of string processing methods in the `str`
 attribute that make it easy to operate on each element of the array, as in the
 code snippet below. Note that pattern-matching in `str` generally uses `regular
-expressions <https://docs.python.org/2/library/re.html>`__ by default (and in
+expressions <https://docs.python.org/3.5/library/re.html>`__ by default (and in
```

Comment:
Can you use `/3/` instead of `/3.5/`?  That way it should always link to the most current version of Python 3.